### PR TITLE
Load tests

### PR DESCRIPTION
**Pull request by grigasp**
Closes https://github.com/iTwin/presentation/issues/133.

Ran against the `BayTownProcessPlant.bim` test dataset that we use in other perf tests using 1, 2 and 4 processes:

| Metric                | 1 process | 2 processes, cache per process | 2 processes, shared cache | 4 processes, cache per process | 4 processes, shared cache |
| --------------------- | --------- | ------------------------------ | ------------------------- | ------------------------------ | ------------------------- |
| http.codes.200        | 20180     | 20184                          | 20184                     | 20188                          | 20187                     |
| http.request_rate     | 60/sec    | 63/sec                         | 57/sec                    | 60/sec                         | 60/sec                    |
| http.requests         | 20180     | 20184                          | 20184                     | 20188                          | 20187                     |
| http.response_time:   |           |                                |                           |                                |                           |
| ...min                | 8         | 8                              | 8                         | 8                              | 8                         |
| ...max                | 3476      | 5016                           | 5016                      | 5055                           | 5023                      |
| ...median             | 18        | 21.1                           | 22.9                      | 24.8                           | 19.1                      |
| ...p95                | 228.2     | 117.9                          | 267.8                     | 104.6                          | 232.8                     |
| ...p99                | 550.1     | 539.2                          | 584.2                     | 550.1                          | 561.2                     |
| http.responses        | 20180     | 20184                          | 20184                     | 20188                          | 20187                     |
| vuser.session_length: |           |                                |                           |                                |                           |
| ...min                | 312818.6  | 308070.4                       | 343598.2                  | 312556.9                       | 321863.2                  |
| ...max                | 321078.8  | 316541.1                       | 353532.1                  | 326717.8                       | 332743                    |
| ...median             | 318179.9  | 311879.3                       | 344681.1                  | 318179.9                       | 324607.8                  |
| ...p95                | 318179.9  | 318179.9                       | 351644.4                  | 324607.8                       | 331165.6                  |
| ...p99                | 318179.9  | 318179.9                       | 351644.4                  | 324607.8                       | 331165.6                  |

I think this clearly shows that our current implementation doesn't scale at all.

### CODE REVIEW

File: apps/test-app/frontend/package.json [hunk 1]
```diff
@@ -35,10 +35,11 @@
     "@itwin/presentation-frontend": "^4.0.0-dev.85",
     "@itwin/webgl-compatibility": "^4.0.0-dev.85",
     "@opentelemetry/api": "^1.4.1",
+    "@types/copy-webpack-plugin": "^8.0.0",
     "@types/node": "^16.0.0",
     "@types/react": "^17.0.37",
     "@types/react-dom": "^17.0.0",
-    "copy-webpack-plugin": "^11.0.0",
+    "copy-webpack-plugin": "^9.0.0",
```
Comment (saskliutas) [inline apps/test-app/frontend/package.json:42]: Why did you need to downgrade `copy-webpack-plugin`?

Comment (grigasp) [reply apps/test-app/frontend/package.json:42]: Some sub-dependency problems... 

`copy-webpack-plugin` [since version 10](https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/CHANGELOG.md#-breaking-changes-1) started using `globby@12`, which [delivers only ESM modules](https://github.com/sindresorhus/globby/releases/tag/v12.0.0). All our other dependencies use `glooby@11`. The two were colliding and causing build failures.

**Review by saskliutas — APPROVED** [review]
